java/api/src/main/java/org/ray/api/Ray.java

```diff
@@ -117,6 +118,24 @@ public static void shutdown() {
     return runtime.wait(waitList, waitList.size(), Integer.MAX_VALUE);
   }
 
+  /**
+   * Wrap a {@link Runnable} with necessary context capture.
+   * @param runnable The runnable to wrap.
+   * @return The wrapped runnable.
+   */
+  public static Runnable asyncClosure(Runnable runnable) {
```

Comment:
This name sounds kind of ambiguous to me. But I don't have a good name either. What about just `wrapRunnable` and `wrapCallable`?

Reply:
Changed to `wrapRunnable` and `wrapCallable`.